src/views/product-integrations-landing/components/searchable-integrations-list/index.tsx

```diff
@@ -247,11 +248,11 @@ export default function SearchableIntegrationsList({
 						/>
 						<MultiSelect
 							text="Types"
-							options={types.map(({ id, name }, i) => {
+							options={types.map(({ id, name, plural_name }, i) => {
 								const selected = typesCheckedArray[i]
 								return {
 									id,
-									label: name,
+									label: plural_name,
```

Comment:
We want the Plural names to display in this dropdown here

<img width="516" alt="CleanShot 2023-05-17 at 16 26 10@2x" src="https://github.com/hashicorp/dev-portal/assets/2105067/0e63fde6-f435-4b07-99cb-3a632dff1f73">